src/startupscriptgenerator/node/scriptgenerator.go

```diff
@@ -119,6 +119,21 @@ func (gen *NodeStartupScriptGenerator) GenerateEntrypointScript() string {
 			commandSource = "PackageJsonMain"
 		}
 
+		if startupCommand == "" {
+			startupCommand = gen.getProcessJsonCommand(userStartupCommandFullPath)
+			commandSource = "ProcessJson"
+		}
+
+		if startupCommand == "" {
+			startupCommand = gen.getConfigJsCommand(userStartupCommandFullPath)
+			commandSource = "ConfigJs"
+		}
+
+		if startupCommand == "" {
```

Comment:
A test in this PR is passing `-userStartupCommand config.yml`...will all these be evaluated or should only the getConfigYaml be executed?...its difficult to follow for me..

Reply:
only getConfigYaml will be executed, since it is the only one that matches the suffix with the test